app/views/home/_find_growers.html.haml

```diff
@@ -0,0 +1,8 @@
+.find-growers.home-banner
+
+  #placesmap
```

Comment:
Please implement the "update in place" now rather than later :)

For the full-width thing, take a look at what we did on http://github.com/3000acres/3000acres with app/views/layouts/home.html.haml (the actual specification of what layout to use happens in the controller).  Basically you just need to make a different layout that doesn't wrap everything in .container.